### PR TITLE
Refactor password policy UI in DomainConfiguration.vue

### PR DESCRIPTION
**Pull request by stephdl**
This pull request refactors the password policy UI in the DomainConfiguration.vue file. The password policy and user portal are displayed only for internal user domain

https://github.com/NethServer/dev/issues/6794

### CODE REVIEW

**Comment by stephdl** [comment]
we need you @andre8244 :D

File: core/ui/src/views/DomainConfiguration.vue [hunk 1]
```diff
@@ -133,82 +133,85 @@
...
-        </cv-column>
+              </template>
+            </NsInfoCard>
+          </cv-column>
+        </template>
         <!-- user portal link -->
         <cv-column :md="4" :max="4">
           <NsInfoCard
             light
+            v-if="domain.location == 'internal'"
```
Comment (andre8244) [inline core/ui/src/views/DomainConfiguration.vue:214]: Let's put this `v-if` into the `<cv-column>` element

File: core/ui/src/views/DomainConfiguration.vue [hunk 2]
```diff
@@ -782,8 +787,10 @@ export default {
       this.domain = taskResult.output.domains.find(
         (d) => d.name == this.domainName
       );
-      this.ListPasswordPolicy();
-      this.getFqdn();
+      if (this.domain.location === "internal") {
```
Comment (andre8244) [inline core/ui/src/views/DomainConfiguration.vue:790]: Let's use a lowercase first letter for `ListPasswordPolicy` function

Comment (stephdl) [reply core/ui/src/views/DomainConfiguration.vue:789]: done

**Review by stephdl — COMMENTED** [review]
I needed to set domain.location to "" else we could have some side effect like display the skeleton until we have the value internal or external

**Review by andre8244 — APPROVED** [review]

File: core/ui/src/views/DomainConfiguration.vue (hunk 1 above)
Comment (stephdl) [reply core/ui/src/views/DomainConfiguration.vue]: done

**Pull request merged**